Project-1/JCheung-P1-README.md

```diff
@@ -0,0 +1,474 @@
+
+(b) ∃x∀y∃z and ∀x∃y∀z 
+
+∃x∀y∃z = "There exists an x such that for every y, there exists a z." This can be interpreted as a statement saying that for some value of x, we can find a corresponding value of z for every possible value of y. It states the existence of x and z in relation to all possible values of y.
+
+∀x∃y∀z = "For all x, there exists a y such that for all z." This can be interpreted as a statement saying that for all possible values of x, we can find a corresponding value of y that holds true for all values of z. In other words, it asserts the existence of y in relation to all possible values of x and z.
+
+(c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+
+∀x∃y∀z∃w = "For all x, there exists a y such that for all z, there exists a w." This can be interpreted as a statement saying that for all possible values of x, we can find a corresponding value of y, and for all possible values of z, we can find a corresponding value of w. In other words, it asserts the existence of y and w in relation to all possible values of x and z.
+
+∃x∀y∃z∀w = "There exists an x such that for all y, there exists a z for all w." This can be interpeted as a statement saying that for some value x, we can find a corresponding value y, and for every y, there is some value z that holds for all of w. 
+	
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```
+
+```mermaid 
+graph LR
+0(0)
+1(1)
+0 ---0
+0 ---1
+1 ---1
+```
+
+U: {0, 1}
+
+R: {(0. 0), (0, 1), {1, 1)}
+
+Interpretation:
+
+On this interpretation, ∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z)) is represented, but ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z)) is not.
+
+On ∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z)), everything should R everything and it is not the case in this graph.
+
+
+	
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+![proof](https://user-images.githubusercontent.com/123851163/216893568-701f0b47-bd92-43fe-9fe7-e6a26d95985a.png)
+
+(b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+
+![proof-2](https://user-images.githubusercontent.com/123851163/216893625-612a4b80-b1e3-4c1c-88fd-98efc3d9285d.png)
+
+(c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+
+![proof-3](https://user-images.githubusercontent.com/123851163/216893677-69d2dacc-a6cd-48e7-a5e2-9539d561bbe2.png)
+
+(d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+
+![proof-4](https://user-images.githubusercontent.com/123851163/216893720-b1b01f6d-a818-46fb-b2c3-243d7903ef9c.png)
+	
+9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+
+```
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+¬(Px ∧ Qx) → (¬Px ∨ ¬Qx)	
```

Comment:
You did not instantiate the universal variable x to a constant. 

You did not provide any justification lines for your proof...